src/game/game2048.ts

```diff
@@ -161,7 +161,12 @@ export interface IGameState {
     return gameEvents;
   }
 
+
+  private exponentialThreshold = 32
   private calculateScoreValue(eventValue: number): number {
+    if (eventValue <= this.exponentialThreshold) {
```

Comment:
Scores were getting too high at lower tile values. Only add exponential scores once we hit a certain threshold.